crates/sui-protocol-config/src/lib.rs

```diff
@@ -2698,10 +2699,15 @@ impl ProtocolConfig {
                     cfg.feature_flags.rethrow_serialization_type_layout_errors = true;
                 }
                 56 => {
+                    if chain == Chain::Mainnet {
+                        cfg.feature_flags.bridge = true;
+                    }
+                }
+                57 => {
                     // Reduce minimum number of random beacon shares.
                     cfg.random_beacon_reduction_lower_bound = Some(800);
                 }
-                57 => {}
+                58 => {}
```

Comment:
Not a major thing if we keep it as-is, but I don't think we need to add a `58` in this PR? Since both the old `56` and `57` should be able to be collapsed into a single protocol version (`57`) since that version hasn't shipped to testnet/mainnet yet.

Reply:
cc @cgswords

Reply:
From the comment I'm guessing this might have been done just to allow for a version execution cut?